deploy.py

```diff
@@ -493,6 +505,40 @@ def clear_apps(self):
       info(("A non-critical error occurred while erasing "
             "apps: {}".format(str(e))))
 
+  def clear_storage(self):
+    if self.args.programmer == "none":
+      return 0
+    board_props = SUPPORTED_BOARDS[self.args.board]
+    storage = bytes([0xFF] * board_props.storage_size)
+    # Use tockloader if possible
+    if self.args.programmer in ("jlink", "openocd"):
+      info("Erasing the persistent storage")
+      tock = loader.TockLoader(self.tockloader_default_args)
+      tock.open()
+      try:
+        tock.flash_binary(storage, board_props.storage_address)
+      except TockLoaderException as e:
+        fatal("Couldn't erase the persistent storage: {}".format(str(e)))
+      return 0
+    # Create an intelhex file otherwise
+    info("Creating the persistent storage HEX file")
+    # pylint: disable=g-import-not-at-top,import-outside-toplevel
+    import intelhex
+    storage_hex = intelhex.IntelHex()
+    storage_hex.frombytes(storage, offset=board_props.storage_address)
+    os.makedirs("target", exist_ok=True)
+    storage_file = "target/{}_storage.hex".format(self.args.board)
+    storage_hex.tofile(storage_file, format="hex")
+    # Flash the intelhex file
+    info("Flashing the persistent storage HEX file")
+    if self.args.programmer == "pyocd":
```

Comment:
pycod supports the `erase` command.
So in this case, it's probably better to have:
```python
self.checked_command([
  "pycod", "erase", "--target={}".format(board_props.pycod_target),
  "--sector {}+{}".format(board_props.storage_address, board_props.storage_size)
])
```

Reply:
Nice. Updated (assuming pycod -> pyocd).